src/bartholomew.rs

```diff
@@ -79,12 +100,24 @@ pub fn render(req: Request) -> Result<Response> {
     engine.load_script_dir()?;
 
     // Load the content.
-    let content_path = content::content_path(PathBuf::from(CONTENT_PATH), &path_info);
+    let content_path;
+    if is_dynamic_template {
```

Comment:
Seems like this could be written something like

```
let content_path = match dynamic_template_params {
  Some((v0, v1)) => // use v0
  None => // use path_info
}
```

?